trunk/SUAVE/Components/Energy/Networks/Battery_Stopped_Rotor.py

```diff
@@ -0,0 +1,170 @@
+# Battery_Stopped_Rotor.py
+# 
+# Created:  Aug 2017, J. Smart
+
+# !! CURRENTLY NON-FUNCTIONAL !!
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+
+# package imports
+import numpy as np
+from SUAVE.Components.Propulsors.Propulsor import Propulsor
+
+from SUAVE.Core import Data
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+class Battery_Propeller(Propulsor):
+    def __defaults__(self): 
+        self.lift_motor        = None
+        self.thrust_motor      = None
+        self.lift_prop         = None
+        self.thrust_prop       = None
+        self.lift_esc          = None
+        self.thrust_esc        = None
+        self.avionics          = None
+        self.payload           = None
+        self.battery           = None
+        self.nacelle_diameter  = None
+        self.engine_length     = None
+        self.num_lift_rotors   = None
+        self.num_thrust_rotors = None
+        self.voltage           = None
+        self.tag               = 'network'
+    
+    # manage process with a driver function
+    def evaluate_thrust(self,state):
+    
+        # unpack
+        conditions      = state.conditions
+        numerics        = state.numerics
+        lift_motor      = self.motor
+        thrust_motor    = self.thrust_motor
+        lift_prop       = self.lift_prop
+        thrust_prop     = self.thrust_prop
+        lift_esc        = self.lift_esc
+        thrust_esc      = self.thrust_esc
+        avionics        = self.avionics
+        payload         = self.payload
+        battery         = self.battery
+        
+        # Set battery energy
+        battery.current_energy = conditions.propulsion.battery_energy  
+
+        # Step 1 battery power
+        lift_esc.inputs.voltagein = state.unknowns.battery_voltage_under_load        
+        thrust_esc.inputs.voltagein = state.unknowns.battery_voltage_under_load
+        # Step 2
+        lift_esc.voltageout(conditions)
+        thrust_esc.voltageout(conditions)
+        # link
+        lift_motor.inputs.voltage = lift_esc.outputs.voltageout
+        thrust_motor.inputs.voltage = thrust_esc.outputs.voltageout 
+        # step 3
+        lift_motor.omega(conditions)
+        thrust_motor.omega(conditions)
+        # link
+        lift_prop.inputs.omega =  lift_motor.outputs.omega
+        lift_prop.thrust_angle = 90
```

Comment:
Change this to radians if possible to avoid confusion later on.

Reply:
Change obsoleted by file deletion.